client/src/App.js

```diff
@@ -1,39 +1,32 @@
-import logo from './logo.svg';
+import React, { useState, useEffect } from 'react';
 import './App.css';
 
 import axios from 'axios';
 
+import YarntypeList from './components/YarntypeList';
+
 function App() {
-  
-  const callServer = () => {
-    console.log("clicked call server button");
-    axios
-      .get("http://localhost:3000/api")
-      .then(res => {
-        console.log('res', res);
-      })
-      .catch(err => {
-        console.log('err', err)
-      })
-  }
+  const [yarntypes, setYarntypes] = useState();
 
-  const getYarn = () => {
-    console.log("attempting to get yarn")
-    axios
-      .get("http://localhost:3000/api/yarntypes")
-      .then(res => {
-        console.log('res', res);
-      })
-      .catch(err => {
-        console.log('err', err)
-      })
-  }
+  useEffect(() => {
+    console.log('Hi, this is useEffect')
+    const getYarntypes = () => {
+      axios
+        .get("http://localhost:3000/api/yarntypes")
```

Comment:
Be thinking about putting the host and port in a config file so that it is not hard coded here. That will make deployments in production much easier since localhost will not be used.